src/main/java/seedu/address/model/meeting/MeetingName.java

```diff
@@ -15,7 +15,7 @@ public class MeetingName {
      * The first character of the address must not be a whitespace,
```

Comment:
```suggestion
     * The first character of the meeting name must not be a whitespace,
```

Could this comment be a typo and should refer to meeting name instead of address?